index.html

```diff
@@ -631,13 +644,32 @@ <h1>Terminology</h1>
       </dt>
       <dd>The combination of Public Security Metadata, Private Security Data, and any other configuration
         information (such as public keys) necessary to operationally configure the security mechanisms of a Thing.</dd>
+      <dt>
+        <dfn>Service</dfn>
+      </dt>
+      <dd>A Service is a software entity that has a network interface.
+      Services can be described by a <a>Thing Description</a> and are a kind of <a>Thing</a>.
+      See also <a>Virtual Thing</a>.
+      Compare with <a>Device</a>.</dd>
       <dt>
         <dfn>Servient</dfn>
       </dt>
       <dd>A software stack that implements the WoT building
         blocks. A Servient can host and expose Things and/or host Consumers that consume Things.
         Servients can support multiple Protocol Bindings to enable
         interaction with different IoT platforms.</dd>
+      <dt>
+        <dfn>Shadow</dfn>
+      </dt>
+      <dd>A Shadow is a <a>Virtual Thing</a> that
+      maintains a copy of the state and mediates interactions with another
+      <a>Thing</a> which may not be immediately available due to network
```

Comment:
I suggest to define a shadow without statements about whether the thing is available or not.
IMHO a shadow reflects the state of another thing.